src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,97 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as nock from "nock";
+import * as api from "../../api";
+
+import { cloudscheduler } from "../../gcp/";
+
+const VERSION = "v1beta1";
+
+const testJob = {
+  name: "projects/test-project/locations/us-east1/jobs/test",
+  schedule: "every 5 minutes",
+  timeZone: "America/Los_Angeles",
+  httpTarget: {
+    uri: "https://afakeone.come",
+    httpMethod: "POST",
+  },
+  retryConfig: {},
+};
+describe("cloudscheduler", () => {
+  let sandbox: sinon.SinonSandbox;
```

Comment:
This can be a `const` and defined here. `sandbox.restore()` takes care of resetting the sandbox, and it doesn't need to be re-created every time

Reply:
Hmmm, I'm realizing that i never actually use sinon here, removing the sandbox and dependency